### PR TITLE
PUB-924: Accessibility timeout

### PR DESCRIPTION
**Pull request by commitBlob**
### Change description ###
As per Accessibility call increasing timeout to 30s 
https://github.com/pa11y/pa11y#timeout-number

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: jest.a11y.config.js [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
+  testRegex: "(/src/test/.*|\\.(test|spec))\\.(ts|js)$",
+  testEnvironment: "node",
   transform: {
     '^.+\\.ts?$': 'ts-jest',
   },
-  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
-  setupFiles: ["<rootDir>/jestEnvVars.js"]
+  moduleFileExtensions: ['ts', 'js'],
+  setupFiles: ["<rootDir>/jestEnvVars.js"],
+  // pa11y default
```
Comment (kurt-uk) [inline jest.a11y.config.js:10]:
```suggestion
  // pa11y recommended default
```
if it was the pa11y default we wouldnt have to set it ;)

**Review by kurt-uk — DISMISSED** [review]

**Review by kurt-uk — APPROVED** [review]

**Review by alberto-hortiguela — APPROVED** [review]
LGTM

**Comment by kurt-uk** [comment]
sorry i thought that this was the recommended default timeout, not the actual default, surely if we dont specify then we should be hitting the 30 second anyway? if the default is 30 seconds and were hitting 5 seconds, then surely we have that coded somewhere? otherwise leaving it blank would mean that we have the 30 second default?

**Comment by commitBlob** [comment]
In case anyone has same question as Kurt we discussed this on the call.
As we use Jest for testing it's default timeout is set to 5 seconds. This overrides pa11y default of 30 seconds, hence in the jest config we define Jest timeout for a11y tests to be 30 seconds.
https://github.com/pa11y/pa11y#timeout-number

**Pull request merged**